ab2d-filters/src/test/java/gov/cms/ab2d/filter/EOBLoadUtilitiesTest.java

```diff
@@ -107,11 +107,11 @@ public void testFacility() {
     }
 
     @Test
-    public void testIdentifier() {
+    void testIdentifier() {
         ExplanationOfBenefit eobSNF = (ExplanationOfBenefit) eobS;
         List<org.hl7.fhir.dstu3.model.Identifier> ids = eobSNF.getIdentifier();
         assertNotNull(ids);
-        assertEquals(ids.size(), 2);
```

Comment:
The syntax is `assertEquals(expected, actual)`

Reply:
Second that !